src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/common/ApplicationLoggerTest.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.companieshouse.chipsrestinterfacesconsumer.common;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
do we need the extend with MockitoExtension and InjectMocks annotations as there isn't anything being mocked

Reply:
it was needed for the injectMocks annotation, but that wan't really needed either so removed them both

Reply:
sorry, you mentioned the injectMocks, only read the MockitoExtension for some reason, anyway sorted now